packages/rest-typings/src/v1/channels/ChannelsListProps.ts

```diff
@@ -6,18 +6,25 @@ const ajv = new Ajv({
 	coerceTypes: true,
 });
 
-export type ChannelsListProps = PaginatedRequest<{ _id?: string; query?: string }>;
+export type ChannelsListProps = PaginatedRequest<{ _id?: string }>;
 
 const channelsListPropsSchema = {
 	type: 'object',
 	properties: {
 		_id: {
 			type: 'string',
-			nullable: true,
 		},
 		query: {
 			type: 'string',
-			nullable: true,
```

Comment:
Why are we removing nullable from query?

Reply:
Actually, it's because these are optional ones. The nullable applies to required ones to allow null. But in these cases we shouldn't allow it 🤔 

I can remove, but as this was a smol change and shouldn't affect (no test, and surely no one sending null to these params) though to keep.

Wdyt?